Monika After Story/game/script-windowreacts.rpy

```diff
@@ -620,3 +620,37 @@ label mas_wrs_word_processor:
     if not wrs_success:
         $ mas_unlockFailedWRS('mas_wrs_word_processor')
     return
+
+init 5 python:
+    addEvent(
+        Event(
+            persistent._mas_windowreacts_database,
+            eventlabel="mas_wrs_crunchyroll",
+            category=[r"(?i)crunchyroll"],
+            rules={
+                "notif-group": "Window Reactions",
+                "skip alert": None,
+                "keep_idle_exp": None,
+                "skip_pause": None
+            },
+            show_in_idle=True
+        ),
+        code="WRS"
+    )
+
+label mas_wrs_crunchyroll:
+    python:
+        if persistent._mas_pm_watch_mangime is None:
+            crunchyroll_quips = ["Oh! So you like anime, [player]?"]
+        else:
```

Comment:
Seeing the condition here, the `else` block handles _both_ whether you said you do like manga/anime _or_ you don't. It might be worth adding another set for not liking

Reply:
> might be worth adding another set for not liking

Hmm, yeah I think you're right.  Not sure what she would say if you loaded crunchyroll after telling her you didn't like anime though.  🤔🤔

Reply:
"I'm glad you're trying to expand your horizons!"

Reply:
Something like "Did you find a series you like?" might be cute.